src/supermarket/spec/authorizers/cookbook_authorizer_spec.rb

```diff
@@ -8,8 +8,13 @@
 
     subject { described_class.new(user, record) }
 
+    context 'and owners can remove cookbooks' do
```

Comment:
"can remove cookbooks when the server is configured to allow it"?

Reply:
Yea, that original phrasing is no bueno. 🤔